axoupdater/src/lib.rs

```diff
@@ -352,8 +351,10 @@ impl AxoUpdater {
         // prefix-style workspaces like CARGO_HOME had the prefix incorrectly
         // set to include the `bin` directory.
         if let Some(provider) = &self.current_version_installed_by {
-            let req = VersionReq::parse(">= 0.10.0-prerelease.1, < 0.15.0-prerelease.8")?;
-            if provider.source == "cargo-dist" && req.matches(&provider.version) {
+            let min = Version::parse("0.10.0-prerelease.1")?;
+            let max = Version::parse("0.15.0-prerelease.8")?;
```

Comment:
please make these `expects`, if they ever fail we want them to fail loud and hard. it should not show up as a silent error in a user's app.